src/components/calcite-sortable-list/calcite-sortable-list.tsx

```diff
@@ -0,0 +1,188 @@
+import Sortable from "sortablejs";
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Prop,
+  State,
+  h
+} from "@stencil/core";
+
+/**
+ * @slot - A slot for adding sortable items
+ */
+@Component({
+  tag: "calcite-sortable-list",
+  styleUrl: "./calcite-sortable-list.scss",
+  shadow: true
+})
+export class CalciteSortableList {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * The class on the handle elements.
+   */
+  @Prop() handleSelector = "calcite-handle";
+
+  /**
+   * When true, disabled prevents interaction. This state shows items with lower opacity/grayed.
+   */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**
+   * When true, content is waiting to be loaded. This state shows a busy indicator.
+   */
+  @Prop({ reflect: true }) loading = false;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteSortableListElement;
+
+  @State() handleActivated = false;
+
+  items: Element[] = [];
+
+  observer = new MutationObserver(() => {
+    this.cleanUpDragAndDrop();
+    this.items = Array.from(this.el.children);
+    this.setUpDragAndDrop();
+  });
+
+  sortable: Sortable;
+
+  handle: Node = null;
```

Comment:
question: Is this being used anywhere? What is the Node type?

Reply:
good catch this is cleanup